CreeDictionary/CreeDictionary/templates/CreeDictionary/paradigm.html

```diff
@@ -32,6 +32,14 @@ <h1 id="head" class="definition-title">
   </header>
 
   {# TODO: .definition_recordings #}
+  <section class="definition__recordings">
+    <template id="template:recordings-explainer">
+      <!-- As I understand it, the text should be rendered here on click, right? -->
+    </template>
```

Comment:
The template should contain the explainer text, but anything inside a `<template>` is NEVER shown on the page! So you have to use JavaScript to breathe life into the template, and place it on the page.

Please see this for a better explanation and an example: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/template


As it stands, nothing uses the `<template>`, so it's just sending wasted bytes to the client :(
If you don't use the template at all in this PR, then the best thing to do is just delete the`<template>` entirely:
```suggestion
```